src/SprykerEco/Zed/Unzer/Business/Refund/UnzerRefundExpander.php

```
@@ -155,6 +155,10 @@ protected function expandUnzerRefundTransfersWithExpenseTransfers(
         RefundTransfer $refundTransfer,
         ExpenseTransfer $expenseTransfer
     ): RefundTransfer {
+        if (!$expenseTransfer->getRefundableAmountOrFail()) {
```

Maybe it would make sense to call `getRefundableAmount()` instead?

---

Would fail in UnzerRefundExpander::createUnzerRefundItemFromExpense at the latest. Imho better fail fast ... ;)